selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FileStructureNodesTest.java

```diff
@@ -156,26 +163,19 @@ public void checkFileStructureNodes() {
     fileStructure.waitExpectedTextInFileStructure(ITEMS_CLASS);
     fileStructure.clickOnIconNodeInFileStructure(JAVA_FILE_NAME);
     fileStructure.waitExpectedTextIsNotPresentInFileStructure(ITEMS_CLASS);
-    fileStructure.clickOnIconNodeInFileStructure(JAVA_FILE_NAME);
-    fileStructure.waitExpectedTextIsNotPresentInFileStructure(ITEMS_INNER_CLASS);
-    fileStructure.waitExpectedTextIsNotPresentInFileStructure(ITEMS_INTERFACE);
-    fileStructure.clickOnIconNodeInFileStructure(INNER_CLASS_NAME);
+    //try-catch was added because of issue: https://github.com/eclipse/che/issues/6499
+    try {
+      fileStructure.clickOnIconNodeInFileStructure(JAVA_FILE_NAME);
```

Comment:
IMHO it 's better to comment on this to explain why we can ignore exception here, and extract repetitive code block to the separate method with meaningful name.